Makefile

```diff
@@ -71,7 +71,7 @@ lint/go:
 # Use shfmt to determine the shell files, takes editorconfig into consideration.
 lint/shellcheck: $(shell shfmt -f .)
 	@echo "--- shellcheck"
-	shellcheck $(shell shfmt -f .)
+	shellcheck -x $(shell shfmt -f .)
```

Comment:
This resolved some warnings around unspecified inputs. See https://www.mankier.com/1/shellcheck#-x and koalaman/shellcheck#902